lib/travis/scheduler/jobs/capacity/base.rb

```diff
@@ -80,6 +82,28 @@ def report(status, job)
             def config
               context.config
             end
+
+            def billing_client
+              @billing_client ||= Billing::Client.new(context)
+            end
+
+            def billing_allowance
+              owner = owners.first
+              owner_class = owner.is_a?(User) ? 'users' : 'organizations'
+
+              billing_client.allowance(owner_class, owner.login)
+            rescue Billing::Client::Error => e
+              if e.response[:status] == 404 # Owner is not on a metered plan
+                return {}
+              end
+
+              raise e
+            end
+            memoize :billing_allowance
+
+            def on_metered_plan?
+              billing_allowance.present?
```

Comment:
New billing system will be active only on https://travis-ci.com/, we don't need to perform the request on https://travis-ci.org/ env (there is no billing service on .org at all)
```suggestion
              return false unless config.com?
              billing_allowance.present?
```